lib/device.js

```diff
@@ -126,6 +160,13 @@ class Device extends EventEmitter {
 						testResult = TestResult.SKIPPED;
 						break;
 					}
+					case RequestResult.STATUS_WAITING: {
+						if (expectingReset) {
+							testResult = TestResult.PASSED;
```

Comment:
When we check the device test result via the status command (ie `s`), it will return `STATUS_WAITING` since it has reset and that is the default state. If we were expecting the reset, then we convert this into a passing test. Right?